Dockerfile

```diff
@@ -121,17 +122,32 @@ RUN npm set progress=false && npm install
 
 RUN pip install -e .
 
+# create static assets
+COPY --chown=app:app ./common/static ./common/static
+RUN mkdir test_root;
+
+ENV EDX_PLATFORM_SETTINGS='assets'
+ENV STATIC_ROOT_BASE='/edx/var/edxapp/staticfiles'
+ENV WEBPACK_CONFIG_PATH='webpack.dev.config.js'
```

Comment:
the value of this variable needs to change depending on whether we are building static assets for dev or prod. Right now it will always build for dev. 

I think our options are either to use a docker build arg to determine if we are building for development, or we could rebuild static assets for production.